Tests/fixtures/mobile_apps.yml

```diff
@@ -1949,3 +1949,35 @@
     model: iPhone
   os_family: iOS
   browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (iPhone; CPU iPhone OS 17_7 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148 ZONApp/iOS/2.5.6;build:3000000110
+  os:
+    name: iOS
+    version: "17.7"
+    platform: ""
+  client:
+    type: mobile app
+    name: ZEIT ONLINE
+    version: "2.5.6"
+  device:
+    type: smartphone
+    brand: Apple
+    model: iPhone
+  os_family: iOS
+  browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (Linux; Android 14; XQ-CC54 Build/65.2.A.2.179; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/128.0.6613.146 Mobile Safari/537.36 ZONApp/Android/2.2.7
+  os:
+    name: Android
+    version: "14"
+    platform: ""
+  client:
+    type: mobile app
+    name: zei ONLINE
```

Comment:
is it missing a "t"? any reason "zei" is in lower case, and not upper one as in another example?

Reply:
Yes sorry about that. For some reason this had reverted to a point I was trying to confirm tests were updated and I didn't catch it.